layers/timeline_semaphore.c

```diff
@@ -2084,7 +2098,7 @@ static VkResult timeline_CreateDevice(
                         fpGetDeviceProcAddr,
                         load_data_info->u.pfnSetDeviceLoaderData,
                         pCreateInfo,
-                        pAllocator ? pAllocator : &instance->alloc,
```

Comment:
Could you expand a bit on this change? I don't understand your comment : 
```
2). Do not use the alloc functions from application, otherwise the
    allocation failure should be checked everywhere.
```

Reply:
There are tests in CTS which failes the allocation intensionally. If the layer uses the CTS's allocation functions, the allocation result must be checked everywhere in this layer and return OUT_OF_MEMORY error. Here is a lazy way to avoid this situation. No applicaiton cares the memory usage of this layer.

Reply:
I can see one case where we don't check the allocation in the layer, but otherwise it seems like it checks in most places. I would rather fix the couple of missing checks.